ci/infra/testrunner/kubectl/kubectl.py

```diff
@@ -67,11 +69,11 @@ def check_kured_lock(self):
             outputformat="-o jsonpath='{.metadata.annotations.weave\.works/kured-node-lock}'").find("manual") != -1
 
 
-    def _run_kubectl(self, command, outputformat='-o json'):
-        shell_cmd = "kubectl --kubeconfig={cwd}/test-cluster/admin.conf \
-                      {format} {command}".format(format=outputformat, command=command, cwd=self.conf.workspace)
+    def run_kubectl(self, command):
```

Comment:
any reason you removed the `outputformat` ? we need it for one of the upgrade tests